doc/ranger.1

```diff
@@ -702,7 +717,7 @@ in ranger.
 .IP "automatically_count_files [bool]" 4
 .IX Item "automatically_count_files [bool]"
 Should ranger count and display the number of files in each directory
-as soon as it's visible?  This gets slow with remote file systems.  Turning it
+as soon as it's visible?  This gets slow with remote file sytems.  Turning it
```

Comment:
A leif in you help text.

Reply:
That wasn't introduced by @mark-dawn I"m pretty sure. It's in master, and I already patched it in one of my PRs.